backend/coreapp/views/compiler.py

```diff
@@ -34,11 +35,12 @@ def compilers_json() -> Dict[str, Dict[str, object]]:
     def platforms_json(
         include_presets: bool = True,
         include_num_scratches: bool = False,
+        profile: Optional[Profile] = None,
     ) -> Dict[str, Dict[str, object]]:
         ret: Dict[str, Dict[str, object]] = {}
 
         for platform in compilers.available_platforms():
-            ret[platform.id] = platform.to_json(include_presets, include_num_scratches)
+            ret[platform.id] = platform.to_json(include_presets, include_num_scratches, profile)
```

Comment:
likewise, I don't think we need any of this plumbing. the presets visible on the site shouldn't change based on who is fetching the data

Reply:
I agree, and will be removing it when I will get some time!

Reply:
Done ✅